### ISSUE TITLE
'suggestions' added to modal window

### ISSUE THREAD

**Issue by AndyatFocallocal** [focallocal/fl-maps#563]
On route 2 users will see a clear and obvious suggestions button.

On route one users go directly into the map. from there if they decide to click '+ resource' they won't see the suggested Public Happiness Activities/Community Support Offers button.

i suggest we add it into the 1st window here, next to the 'next' button

![image](https://user-images.githubusercontent.com/16229696/40362976-fff5f700-5dc5-11e8-87a0-c2b456543ff5.png)

on focallocal.org it links here: http://activities.focallocal.org/

on brightertomorrowmap.com it links here: https://www.reddit.com/r/brightertomorrow/

**Comment by royGil** [comment]
Sorry I didn't understand what you are asking for

**Comment by AndyatFocallocal** [comment]
we discussed this further on Slack, i don't think i included my reasons for using Reddit on the Brighter Tomorrow Map. 

1) It works. No development needed, just adding a link
2) Marketing. Our map can reach their huge community while it's growing it's own

Everytime i've posted about the map on Reddit we've ended up on the front page, so having a reddit thread will be a fabulous way to reach and retain those ten's of thousands of people's interest. 

Happy to revisit it later on and discuss bringing it in house, for now it's quick and will help grow our own community
